controlplane/modules/route/mod.go

```diff
@@ -119,7 +127,7 @@ func (m *RouteModule) Run(ctx context.Context) error {
 	}
 
 	for {
-		if _, err := gateway.Register(ctx, req); err == nil {
+		if _, err = gateway.Register(ctx, req); err == nil {
```

Comment:
Why?

Reply:
`gopls` suggests these changes; it shows a warning about variable shadowing.